### DISCUSSION TITLE
Question related IFC export and custom properties

### DISCUSSION THREAD

**Discussion by Nhillus** [iTwin/itwinjs-core#6715] [Q&A]
Hi! I've successfully added some new properties to my iModel and am currently displaying these custom properties in an iTwin visualizer. However, when I try to export these properties to an IFC file, they are not included. Could it be that I need an additional library or tool to handle the export of custom properties, makea  custom export ?

**Comment by pratikshan85** [comment]
@Nhillus Can you please provide more details about your custom properties and iModel also?
